specification/iotcentral/resource-manager/Microsoft.IoTCentral/stable/2021-06-01/examples/Apps_CreateOrUpdate.json

```diff
@@ -65,6 +73,11 @@
         },
         "sku": {
           "name": "ST2"
+        },
+        "identity": {
```

Comment:
I think there will be three different scenarios for the response

1. No "identity": {} since it was never set.
2. Identity with type of SystemAssigned.
3. Identity with type of None.

We should show that in the example.

Reply:
Is there precedence or other example showing all different scenarios? I noticed enum for AppState (created/suspended) does not provide all scenarios in the examples (only created in shown in examples).

Reply:
The response for these 3 examples are vastly different. That is why I think it may be a good idea to include it in our examples. Just something nit. Up to you to decide.